### PR TITLE
EDU-733 Always render media in block markdown inputs

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-733

### CODE REVIEW

File: src/components/markdown.js [hunk 1]
```diff
@@ -4,30 +4,30 @@ import { useService } from './container-context.js';
...
 import GithubFlavoredMarkdown from '../common/github-flavored-markdown.js';
 
-function Markdown({ className, children, inline, renderMedia, tag, ...rest }) {
+function Markdown({ className, children, inline, tag, ...rest }) {
   const gfm = useService(GithubFlavoredMarkdown);
   const { cdnRootUrl } = useService(ClientConfig);
 
   const Tag = tag || 'div';
+  const renderMethod = inline ? 'renderInline' : 'render';
+  const innerHtml = { __html: gfm[renderMethod](children, { cdnRootUrl, renderMedia: !inline }) };
```
Comment (adrianaluchian) [inline src/components/markdown.js:13]: I left here the only `renderMedia` switch ☝️

Comment (ahelmberger) [reply src/components/markdown.js:13]: :+1:

**Comment by ahelmberger** [comment]
We should also update `markdown-help.yml` where it says:

> Also, in some places, where explicitly indicated by the option “render media”, you can embed video or audio files ...

**Comment by adrianaluchian** [comment]
> We should also update `markdown-help.yml` where it says:
> 
> > Also, in some places, where explicitly indicated by the option “render media”, you can embed video or audio files ...

Good catch, updated hopefully correctly.

File: src/components/markdown-help.yml [hunk 2]
```diff
@@ -158,7 +158,7 @@ blockHelpMarkdown:
 
     Bitte beachte, dass Link- bzw. Bildadressen keine Leerzeichen enthalten dürfen, um korrekt dargestellt zu werden.
 
-    Darüberhinaus kannst du auf die gleiche Weise in einigen Feldern (gekennzeichnet mit der Option „Medien wiedergeben“), anstelle von Bildern auch Video- bzw. Audio-Dateien einbinden.
+    Darüberhinaus kannst du auf die gleiche Weise von Bildern auch Video- bzw. Audio-Dateien einbinden.
```
Comment (ahelmberger) [inline src/components/markdown-help.yml:161]: "anstelle" is missing between "Weise" and "von"

Comment (adrianaluchian) [reply src/components/markdown-help.yml]: Re-corrected, thanks!

Comment (ahelmberger) [reply src/components/markdown-help.yml]: Cool!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**